spec/models/post_spec.rb

```diff
@@ -1,5 +1,17 @@
 require 'rails_helper'
 
 RSpec.describe Post, type: :model do
-  
+  context 'validation post test' do
+    it 'ensured body presence' do
+      post = Post.new(body: 'good').save
```

Comment:
Do not try to save it if you don’t check the reason of falsey  result

I would definitely recommend you to check that record is invalid

For that you just need to initialize it with `new(...)` and then call `invalid?`

Fortunately rspec can do it for us